tasks/linux.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+action=$PT_action
```

Comment:
More of a question than a comment

Where do PT_action / PT_name originate?
I'm assuming they're PuppetTask_action / name?